src/types.ts

```diff
@@ -1455,6 +1456,7 @@ export type Event = CustomEventData & {
   connection_id?: string;
   // event creation timestamp, format Date ISO string
   created_at?: string;
+  deleted_for_me?: boolean;
```

Comment:
Is it necessary to add this property to the WS event payload root if it is already contained inside the `message` object? I suppose that this would be message.deleted event that carries the `message`.

Reply:
I see that I already pointed this out and see that the `message` inside the WS event is not getting the property.